tests/integration.rs

```diff
@@ -7337,7 +7337,18 @@ mod live_migration {
     // 4. The destination VM is functional (including various virtio-devices are working properly) after
     //    live migration;
     // Note: This test does not use vsock as we can't create two identical vsock on the same host.
-    fn _test_live_migration(upgrade_test: bool, numa: bool, local: bool, watchdog: bool) {
+    fn _test_live_migration(
+        upgrade_test: bool,
+        numa: bool,
+        local: bool,
+        watchdog: bool,
+        balloon: bool,
+    ) {
+        assert!(
+            !(numa && balloon),
+            "Invalid inputs: 'numa' and 'balloon' cannot be tested at the same time."
+        );
```

Comment:
I think it's partially supported, but I agree let's not introduce too much complexity here. And the thing is, I'm not sure how inflating the balloon would impact one NUMA node or another.

Reply:
Yes, I decided to put this guard for this exact reason. We can revisit this later if needed.